### PR TITLE
Full orderbook update

### LINKED ISSUE #151: Full Orderbook
Currently Orderbook only shows 8 order prices closest to the market. We need to be able to show the whole orderbook somehow, maybe by aggregating the prices (with mouse scroll or via a selector somewhere on top of the orderbook).

### PR DESCRIPTION
**Pull request by nguvictor**
Fixes #151

### CODE REVIEW

**Review by EvgeniiaVak — CHANGES_REQUESTED** [review]
Also please remove the console logs (see eslint warnings).

Does SDK provide full orderbook out of the box? I think I've a discussion about it somewhere, but now I can't find it.

File: src/app/redux/orderBookSlice.ts [hunk 1]
```diff
@@ -30,11 +31,13 @@ const initialState: OrderBookState = {
...
 };
 
 export function toOrderBookRowProps(
   adexOrderbookLines: adex.OrderbookLine[],
-  side: "sell" | "buy"
+  side: "sell" | "buy",
+  grouping: number
 ): OrderBookRowProps[] {
   // this will drop the rows that do not fit into 8 buys/sells
   // TODO: implement pagination or scrolling
```
[the remark this answers is not in the document]
Comment (nguvictor) [reply src/app/redux/orderBookSlice.ts]: @fliebenberg does the SDK provide anyway to aggregate or group the orderbook?

[the remark this answers is not in the document]
Comment (fliebenberg) [reply src/app/redux/orderBookSlice.ts]: > @fliebenberg does the SDK provide anyway to aggregate or group the orderbook?

Nope.

[the remark this answers is not in the document]
Comment (EvgeniiaVak) [reply src/app/redux/orderBookSlice.ts]: @fliebenberg does SDK have some limit on how many orders are fetched by default?

[the remark this answers is not in the document]
Comment (fliebenberg) [reply src/app/redux/orderBookSlice.ts]: > @fliebenberg does SDK have some limit on how many orders are fetched by default?

Not at the moment. Will be adding that in near future. Will probably limit it to 30 by default.

[the remark this answers is not in the document]
Comment (EvgeniiaVak) [reply src/app/redux/orderBookSlice.ts]: Ok, then this PR functionality works correctly for now. @fliebenberg please let us know when the limit is implemented, we will need to update orderbook to fetch more data for the grouping.

**Review by EvgeniiaVak — APPROVED** [review]

**Pull request merged**